_data/pubs.yml

```diff
@@ -1,3 +1,21 @@
+
+- id: Ellis2020
+  title: "Facilitating open-science with realistic fMRI simulation: validation and application"
+  date: 2020-02-19
+  authors:
+     - Cameron T. Ellis
+     - Michael Lesnick
+     - Gregory Henselman-Petrusek
+     - Bryn B. Keller
+     - Theodore L. Willke
+     - Jonathan D. Cohen
```

Comment:
These are the authors of Ellis2019.

Reply:
Goodness, those were some bad mistakes, thanks for catching that